.github/workflows/verify_examples.yml

```diff
@@ -0,0 +1,73 @@
+# Created with package:mono_repo v3.4.1
+name: Verify Examples
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+  schedule:
+  - cron: "0 0 * * 0"
+defaults:
+  run:
+    shell: bash
+env:
+  PUB_ENVIRONMENT: bot.github
+
+jobs:
+  verify_examples:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        example_directory: ["examples/hello"]
```

Comment:
@subfuzion – you can just add every example to this variable – assuming they all have the same shape – and they'll get tested!

Reply:
What if they have a different shape, like `cloudevent` handlers? What needs to get updated for that?

Reply:
Not sure I understand what you mean by that. Looks like the matrix has to be updated for each `example/<x>` directory and tests for the example have to pass. Is there something else?

Reply:
That's true. It has to be updated. But it's one entry in one place in one file to get all of the verification we want – without adding little 💩  in every directory